main.js

```diff
@@ -144,6 +145,11 @@ module.exports = function(options) {
 		next();
 	});
 
+	app.use(function(req, res, next) {
+		res.locals.jsonLd = [nextJsonLd.webSite()];
+		next();
+	});
```

Comment:
do we want to make this an 'opt in' option - https://github.com/Financial-Times/n-express/blob/0ce7743c8a0d768a717b090a2fd59f04f855fc06/main.js#L53-L64